### PR TITLE
Fix Croatian mask

### PR DESCRIPTION
**Pull request by charlesmulder**
Thank you for the fix relating to #1359

I have tested with these Croatian number variations:

+385 91 237 8276
+385 23 265 461
+385 1 4898 555

and applied a fix.

### CODE REVIEW

File: extra/phone-codes/phone.js [hunk 1]
```diff
@@ -127,8 +127,8 @@
 				{ "mask": "+592-###-####", "cc": "GY", "cd": "Guyana", "desc_en": "", "name_ru": "Гайана", "desc_ru": "" },
 				{ "mask": "+852-####-####", "cc": "HK", "cd": "Hong Kong", "desc_en": "", "name_ru": "Гонконг", "desc_ru": "" },
 				{ "mask": "+504-####-####", "cc": "HN", "cd": "Honduras", "desc_en": "", "name_ru": "Гондурас", "desc_ru": "" },
-				{ "mask": "+385-##-###-###", "cc": "HR", "cd": "Croatia", "desc_en": "", "name_ru": "Хорватия", "desc_ru": "" },
-				{ "mask": "+385-1-###-###", "cc": "HR", "cd": "Croatia", "desc_en": "", "name_ru": "Хорватия", "desc_ru": "" },
+				{ "mask": "+385-(##)-###-###[#]", "cc": "HR", "cd": "Croatia", "desc_en": "", "name_ru": "Хорватия", "desc_ru": "" },
```
Comment (RobinHerbots) [inline extra/phone-codes/phone.js:130]: @charlesmulder , 

Can you split this mask in 2 separate masks.  Otherwise the metadata retrieval will fail.

Comment (charlesmulder) [reply extra/phone-codes/phone.js:130]: Sure. Thanks. I'll do it today.

Comment (charlesmulder) [reply extra/phone-codes/phone.js:130]: Hi Robin,

Just checked and I did split it into two masks as follows:

```
{ "mask": "+385-(##)-###-###[#]", "cc": "HR", "cd": "Croatia", "desc_en": "", "name_ru": "Хорватия", "desc_ru": "" },
{ "mask": "+385-1-####-###", "cc": "HR", "cd": "Croatia", "desc_en": "", "name_ru": "Хорватия", "desc_ru": "" },
```

which seems to work.

Comment (charlesmulder) [reply extra/phone-codes/phone.js:130]: The diff is misleading.

**Pull request merged**